ui/.eslintignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+out
+.next
+static
+package.json
+package-lock.json
+nodemon.json
```

Comment:
It would be great if there was a newline character at end of file and was enforced via the `eol-last` eslint rule.